cmd/f3/msgdump/envelope.go

```diff
@@ -39,7 +39,7 @@ type PJustification struct {
 type PPayload struct {
 	Instance         uint64
 	Round            uint64
-	Step             string `parquet:"Step,enum"`
+	Phase            string `parquet:"Phase,enum"`
```

Comment:
Note this is a breaking change if we ever want to analyse old parquet logs.

Reply:
The analysis is done in duckdb, so the queries will have to be updated, but it is fine.